tests/test_find_iocs.py

```diff
@@ -37,7 +37,7 @@ def _compare_lists(ioc_list: IndicatorList, result_list: IndicatorList) -> None:
 
 
 def _compare_dicts(ioc_dict: IndicatorDict, result_dict: IndicatorDict) -> None:
-    for key in ioc_dict.keys():
+    for key in result_dict.keys():
```

Comment:
I think this change is incorrect b/c on the assertion on the [next line](https://github.com/fhightower/ioc-finder/pull/202/files#diff-327c38d4b712e5b40771cd515d11d0673945fea4a8091e2876ba9d1d026ee10cR41) doesn't make sense in light of this change.

What's the intended purpose of this change? From what I can tell, the code would work just fine if it were left as it was:

```suggestion
    for key in ioc_dict.keys():
```